source/__snapshots__/default-highway-test.snap

```diff
@@ -1120,16 +1120,108 @@ export interface Priority {
   __entity_type__?: "Priority";
   __permissions?: Record<string, any>;
 }
-export type Milestone = TypedContext<"Milestone">;
-export type Task = TypedContext<"Task">;
-export type Episode = TypedContext<"Episode">;
-export type Scene = TypedContext<"Scene">;
-export type Folder = TypedContext<"Folder">;
-export type Image = TypedContext<"Image">;
-export type AssetBuild = TypedContext<"AssetBuild">;
-export type Shot = TypedContext<"Shot">;
-export type Sequence = TypedContext<"Sequence">;
-export type Information = TypedContext<"Information">;
+export type Milestone = Omit<TypedContext<"Milestone">, "custom_attributes"> & {
```

Comment:
More of a long shot, but it could be possible to use a map here as well to add the custom attributes straight in `TypedContext` and not have to create new types.

Reply:
It would get rid of having to create new types and have both `TypedContext<"Milestone">` and `Milestone` with only subtle difference

Reply:
Don't have time to develop it fully, but something like this is what I'm imagining:
```
export interface TypedContextSubtypeMap {
  TypedContext: TypedContext;
  Milestone: Milestone;
}

export interface TypedContextCustomAttributesMap {
  TypedContext: Array<
    TypedCustomAttributeValue<"somedynamicenumerator">
    | TypedCustomAttributeValue<"somedate">
    | TypedCustomAttributeValue<"sometext">
  >;
  Milestone: Array<
    TypedCustomAttributeValue<"somedynamicenumerator">
  >;
}

export type TypedContextSubtype = keyof TypedContextSubtypeMap;

// Change TypedContext to have the new custom_attributes
export interface TypedContext<
  K extends TypedContextSubtype = TypedContextSubtype
> extends Omit<Context, "__entity_type__" | "__permissions" | "custom_attributes"> {
  __entity_type__: K;
  custom_attributes: TypedContextCustomAttributesMap[K]; // Not sure if this would be accepted, or we would need to change "K extends TypedContextSubtype = TypedContextSubtype" to something else. Not sure there is a way to tell TypeScript that the keys of two maps are the same.
  // [Rest of the properties from Context that are not omitted]
}
```

As I mentioned I'm not even sure it's possible and becomes a bit more involved, but if it sparks your imagination you can see if you can make something out of it.

Reply:
This has now been resolved.